Sierra/challenge_14/book_list_class.py

```diff
@@ -0,0 +1,37 @@
+from book_class import Book
+
+
+class Booklist():
+	def __init__(self):
+		"""Initialize the empty book list"""
+		self.books = []
+
+	def add(self, title, author):
+		"""Add a Book object with the given title and author to the book list"""
+		book = Book(title, author)
+		self.books.append(book.title)
+		self.books.append(book.author)
+		# I had it formatted as: self.books.append(f'{book.title} - {book.author}') but then 
+		# I couldn't remove the title if it was in self.books for the remove_title method
+		
+	def count_books(self):
+		"""Return the number of books currently in the booklist"""
+		print(len(self.books))
+
+	def remove_title(self, title):
+		"""Remove a book from the book list"""
+		if title in self.books:
+			self.books.remove(title)
```

Comment:
For the original design, you can loop through all the books, and then call  if book.title == title, then remove the book object. Something like this:
for book in self.books:
    if book.title == title: 
         self.book.remove(book)